tests/CommunityToolkit.Aspire.Hosting.EventStore.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using CommunityToolkit.Aspire.Testing;
+using FluentAssertions;
+using Projects;
+using System.Net.Http.Json;
+
+namespace CommunityToolkit.Aspire.Hosting.EventStore.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<CommunityToolkit_Aspire_Hosting_EventStore_AppHost> fixture) : IClassFixture<AspireIntegrationTestFixture<CommunityToolkit_Aspire_Hosting_EventStore_AppHost>>
+{
+    [Fact]
+    public async Task ResourceStartsAndRespondsOk()
+    {
+        var resourceName = "eventstore";
+        await fixture.ResourceNotificationService
+            .WaitForResourceAsync(resourceName, KnownResourceStates.Running)
```

Comment:
it's better to use the method that waits for the resource to be healthy, so consideration is taken for the app in the container starting up

Reply:
Addressed in the last commit: https://github.com/CommunityToolkit/Aspire/pull/277/commits/7deb7d5bc32aa830d8266342076c010509f75b88